app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] When I try to sign up as a new user I have the error on the image below, Please fix that.

![image](https://github.com/hamidazim321/blog_app/assets/68796232/daf687a2-438e-4175-9cc7-7a1f43b9db1c)